src/platforms/96b_carbon/platform.h

```diff
@@ -100,9 +97,9 @@
  * For now USART preempts USB which may spin while buffer is drained.
  * TIM3 is used for traceswo capture and must be highest priority.
  */
-#define IRQ_PRI_USB		(2 << 4)
-#define IRQ_PRI_USBUSART	(1 << 4)
-#define IRQ_PRI_USBUSART_TIM	(3 << 4)
+#define IRQ_PRI_USB		(1 << 4)
```

Comment:
Could you update the comment above as well so it reflects the new prioritisations?

Reply:
Good catch. Thank you. I have updated the comment. Let me know if it is correct/sufficient. :)